slave-services/src/main/java/org/iqu/slaveservices/rest/consumer/NewsConsumer.java

```diff
@@ -9,11 +9,11 @@
 
 public interface NewsConsumer {
 
-	NewsDTO retrieveNews(ServiceInfo serviceInfo);
+  public NewsDTO retrieveNews(ServiceInfo serviceInfo);
```

Comment:
There's no need to add public since interface methods are always public, so please remove them